client/src/components/feeds/post/comments/comment_header/CommentMenuButton.css

```diff
@@ -85,6 +85,9 @@
   box-shadow: -0.1rem 0.1rem 0rem 0.03rem rgba(0, 0, 0, 1);
 }
 
+[data-reach-dialog-content][aria-label="AnonymousConfirmationBox"] {
+  width: 25% !important;
```

Comment:
This looks like an improvement for desktop, but on mobile it looks a lot more squished than it did before. Might want to address the mobile styling of this in some way.